### DISCUSSION TITLE
Softaculous installation is broken

### DISCUSSION THREAD

**Discussion by DaddyApi** [firefly-iii/firefly-iii#7036] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely me, not you](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

It's not possible to install the app via Softaculous. The installation succeeds but the app is not working.
Tried different hosting no luck.

### Expected behaviour

Install and run successfully

### Debug information

The further I went was this:

```
Whoops! An error occurred.

Unfortunately, this error was not recoverable :(. Firefly III broke. The error is:
Could not access the database: could not find driver (SQL: select count(*) as aggregate from `users`)
Error
This error occured in file /home/redacted/fin/app/Http/Middleware/Installer.php on line 108 with code 0. 
```
this error indicates that there is a problem with the connector (not installed) though, I'm 100% sure everything from requirements is installed

### Steps to reproduce

Steps to reproduce:
1) Get a hosting
2) Set PHP version 8.2
3) Enable PHP modules:

    PHP BCMath Arbitrary Precision Mathematics
    PHP Internationalization extension
    PHP Curl
    PHP Zip
    PHP Sodium
    PHP GD
    PHP XML
    PHP MBString
    PHP whatever database you're gonna use.
4) Install via Softaculous

**Comment by JC5** [comment]
I'm sorry, you'll have to take this up with either your provider or Softaculous.

**Reply by DaddyApi** [reply]
> I'm sorry, you'll have to take this up with either your provider or Softaculous.

I don't think it's softaculous only. I tried to pull from git and build. Same situation. Looks like major bug, because I can reproduce it on clean new hosting provider. The providers also have the feeling that this is a bug. I have a hosting on FastComet, and their support is pretty nice, they tried a clean install and softaculous install, same situation.

**Reply by JC5** [reply]
Alright. Share your command history and .env file please. I'm happy to take a look.
